sdk/identity/azure-identity/pom.xml

```diff
@@ -115,6 +115,13 @@
       <version>1.17.7</version> <!-- {x-version-update;testdep_net.bytebuddy:byte-buddy-agent;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <!-- BouncyCastle is required for generating self-signed certificates with SANs in tests -->
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-lts8on</artifactId>
```

Comment:
Bouncy castle isn't a great dependency to rely on, due to CVE exposure. 
There should be alternative approaches to do this.